### DISCUSSION TITLE
Converting to Prisma

### DISCUSSION THREAD

**Discussion by N1GHTR4NG3R** [prisma/prisma#9395] [Q&A]
Having a hard time converting to Prisma, I can't understand how I would convert my mySql code for Prisma CLI.

I have this bit of code, which checks the current table, and matches the references already, but then if the database is out, It would update with the correct values:
```
async function guildInfo() {
    try {
      let guildDB = bot.guilds.cache.forEach((g) => {
        let gInfo = {
          guild_id: g.id,
          guild_name: g.name,
          guild_owner: g.owner.user.username,
          guild_owner_id: g.ownerID,
          guild_member_count: g.memberCount,
          guild_prefix: g.prefix,
        };
        let gSql = "INSERT IGNORE INTO Guilds Set ?";
        bot.con.query(gSql, gInfo, (err, result) => {
          if (err) {
            console.error(
              "Unable to update Guild data Table: Index.js - Line 58"
            );
          }
          return result;
        });
      });
      guildDB;
```

I can only presume it would be on the ```updateMany```  event, but it isn't making much sense to me on converting it to the CLI syntax. If anyone is able to help, I would appreciate this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
